changelog.md

```diff
@@ -96,6 +96,13 @@
 with other backends. see #9125. Use `-d:nimLegacyJsRound` for previous behavior.
 - Added `socketstream` module that wraps sockets in the stream interface
 
+- Changed the behavior of `uri.decodeQuery` when there are unencoded `=`
```

Comment:
> I'm not convinced that the CI errors comes from my code, errors seems to come from fidget and PackageFileParsed (whatever those are) and I don't really see the problem looking at the CI logs...

https://dev.azure.com/nim-lang/255dfe86-e590-40bb-a8a2-3c0295ebdeb1/_apis/build/builds/12009/logs/84

2021-01-11T23:28:40.1530530Z [1m[31mFAIL: [36mtests/stdlib/turi.nim c[0m
...
2021-01-11T23:28:40.1541230Z ../../lib/system/fatal.nim(53, 5) Error: unhandled exception: expected raising 'UriParseError', instead nothing was raised by: 
2021-01-11T23:28:40.1542060Z discard toSeq(decodeQuery("a=1&b=2c=6")) [AssertionDefect]

it comes from this PR, the fix is trivial:
adapt:
```nim
    doAssertRaises(UriParseError):
      discard toSeq(decodeQuery("a=1&b=2c=6"))
```
accordingly

Reply:
Thank you, I was looking at the other CI runs where this did not appear, my bad.

Reply:
no problem; please click "resolve conversation" for any resolved comment (unless it contains pushback)